tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -13,50 +13,142 @@
     preparsing_validators=[
         validators.notEmpty(start=19, end=60),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='SSN', result_function=validators.notOneOf([str(i)*9 for i in range(0, 9)]),
```

Comment:
it appears that this range excludes 9. should be inclusive. all 9s is quite common. 

![2369p3](https://github.com/raft-tech/TANF-app/assets/63075587/5d00a9ad-f8c8-4b39-ba6a-0f185cd76771)

Reply:
also it does not appear that theses validators are checking the possible 2nd child's data in the T3 record.

![2369p4](https://github.com/raft-tech/TANF-app/assets/63075587/e972095a-5330-47f1-aabd-3b413075d025)

Reply:
just noticed that the 2nd child's validators are further below.  under what condition does both sets of cat2 and cat3  validators run? not run?

Reply:
The validators run if if the fields are populated. So the validators for the second child would run if the T3 record did have a second child. The SSN error you showed was an issue in our range validator which you found and called out below. Fixed now :).

Reply:
Added the 9 by extending the range to range(0, 10)